hw/xbox/smbus_xbox_smc.c

```diff
@@ -189,6 +223,14 @@ static void smbus_smc_realize(DeviceState *dev, Error **errp)
     if (object_property_get_bool(qdev_get_machine(), "short-animation", NULL)) {
         smc->scratch_reg = SMC_REG_SCRATCH_SHORT_ANIMATION;
     }
+
+    avpack = object_property_get_str(qdev_get_machine(), "avpack", NULL);
```

Comment:
nit: `assert(avpack != NULL);` to guard against `NULL` case (we don't think it can happen; but better safe than sorry).

Reply:
Added.